recipe/meta.yaml

```
@@ -46,11 +46,13 @@ requirements:
     - libpng
     - hdf5 * {{ mpi_prefix }}_*
     - zlib
+    - numpy 1.16
```

We already [require `numpy` version `1.16` globally]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/fdd3b84736a19675266fa9d400d6a1580d5eba33/recipe/conda_build_config.yaml#L546-L547 ). So it need not be specified here.

```suggestion
    - numpy
```